R/markdown.R

```diff
@@ -121,10 +117,33 @@ convert_markdown_to_html <- function(in_path, out_path, ...) {
       "--indented-code-classes=R",
       "--section-divs",
       "--wrap=none",
-      "--mathjax",
+      paste0("--", config_math_rendering(pkg)),
```

Comment:
I haven't tested this, but when looking over the code, I think `config_math_rendering(pkg)` gives just the `template.math-rendering` config value (`mathml`, `mathjax` or `katex`). But for `katex` and `mathjax`, we [need to provide Pandoc also the path to the asset](https://pandoc.org/MANUAL.html#math-rendering-in-html), i.e. `--mathjax=PATH` / `--katex=PATH`, otherwise Pandoc will generate CDN links.

Reply:
Does that matter? I don't think we think we use anything that pandoc puts in `<head>`.

Reply:
Err, I don't know... if we are really ignoring the `<head>` output completely, it shouldn't matter and you're right, I guess.